app/i18n/index.js

```diff
@@ -30,6 +30,11 @@ function loadTranslation(locale) {
             localeData = require('react-intl/locale-data/de');
             momentData = require('moment/locale/de');
             break;
+        case 'en-AU':
+            translations = require('@assets/i18n/en-AU.json');
```

Comment:
@enahum Should this match actual file under assets? `assets/base/i18n/en_AU.json` looks like file has underscore and not dash. But for prod it may be injecting a different file?

Reply:
this line is fine.. just the file under the assets folder is wrong